Automattic/jetpack
# Manage: Remove Plugin Update Cards from Settings and Dashboard

Fixes #16933 

#### Changes proposed in this Pull Request:

* Removes the plugin update card (for auto-updating plugins) from both the Settings and Dashboard pages.

##### Settings

Before | After
--- | ---
<img width="1065" alt="Screen Shot 2020-08-21 at 1 33 28 PM" src="https://user-images.githubusercontent.com/1689238/91498871-8b66ed80-e88e-11ea-8aee-1fc6b6f42b4d.png"> | <img width="1060" alt="Screen Shot 2020-08-21 at 1 32 57 PM" src="https://user-images.githubusercontent.com/1689238/91498870-8ace5700-e88e-11ea-879b-a4a7d7db5eb0.png">

##### Dashboard

Before | After
--- | ---
<img width="1063" alt="Screen Shot 2020-08-27 at 5 53 29 PM" src="https://user-images.githubusercontent.com/1689238/91499074-d8e35a80-e88e-11ea-8951-5a590d3f1cf4.png"> | <img width="1059" alt="Screen Shot 2020-08-27 at 5 47 43 PM" src="https://user-images.githubusercontent.com/1689238/91498965-ac2f4300-e88e-11ea-993e-e24df2bfc389.png">


#### Jetpack product discussion


[Plugin auto-updates were added in WordPress 5.5](https://make.wordpress.org/core/2020/07/15/controlling-plugin-and-theme-auto-updates-ui-in-wordpress-5-5/)

#### Does this pull request change what data or activity we track or use?


No changes.

#### Testing instructions:






* Go to '/wp-admin/admin.php?page=jetpack#/settings'
- [ ] There should be no auto-update plugins card
* Go to '/wp-admin/admin.php?page=jetpack#/dashboard'
- [ ] There should be no auto-update plugins card

#### Proposed changelog entry for your changes:

* Remove plugin update cards from Settings and Dashboard pages
